source/cluster_manager/cluster_manager.toml

```diff
@@ -4,6 +4,10 @@ port = 5672 #default: 5672
 
 [manager]
 name = "owt-cluster"
+# how many raft node in this cluster,raft will select leader when the candidate got votes gt totalNode/2
+totalNode = 3
```

Comment:
It can be set to 1, but if more than one cluster is started, when the leader of the first cluster goes offline, a brief split-brain and data loss may occur during the re-election process. In cases of weak network connections, the clusters may also constantly alternate as the master.

Reply:
It can be set to 1, but if more than one cluster is started, when the leader of the first cluster goes offline, a brief split-brain and data loss may occur during the re-election process. In cases of weak network connections, the clusters may also constantly alternate as the master.

Reply:
> It can be set to 1, but if more than one cluster is started, when the leader of the first cluster goes offline, a brief split-brain and data loss may occur during the re-election process. In cases of weak network connections, the clusters may also constantly alternate as the master.
It can be set to 1, but if more than one cluster is started, when the leader of the first cluster goes offline, a brief split-brain and data loss may occur during the re-election process. In cases of weak network connections, the clusters may also constantly alternate as the master.

Reply:
If we start single owt-server cluster by `bin/start-all.sh`, will it work properly with this configuration?

Reply:
> If we start single owt-server cluster by `bin/start-all.sh`, will it work properly with this configuration?

jt can't elect leader require got votes>totalNode/2，if you change the default value totalNode=1，it work properly

Reply:
Your work is great, but we need to ensure that it can successfully start and run under the default configuration. If you have confirmed it works well when `totalNode` is 1, change it to 1 please.

Reply:
> You can add description about this in comment. like "Set its value to the number of cluster manager instances". So users can edit it when they needed.

ok